tests/test_app.py

```diff
@@ -76,7 +76,6 @@ def test_app_configure_with_platforms(mocked_engine):
         token='should-be-a-valid-token'
     )
     mocked_engine['instance'].configure.assert_called_with()
-    mocked_engine['instance'].tasks.assert_called_with()
```

Comment:
We should make sure that tasks is a list and it returns the right tasks, in this case, polling.